panel/applets/icon-tasklist/WindowListItem.vala

```diff
@@ -0,0 +1,138 @@
+/*
+ * This file is part of budgie-desktop
+ *
+ * Copyright (C) 2016 Fernando Mussel <fernandomussel91@gmail.com>
```

Comment:
So, for this file that includes code from both of us, I would also need to credit authorship to both of us. So is there a specific standard that the project is using? For instance, the oldest copyright comes first:

Copyright (C) 2014-2016 Ikey Doherty <ikey@solus-project.com>
Copyright (C) 2016 Fernando Mussel <fernandomussel91@gmail.com>